docs/adguard-for-android/solving-problems/compatibility-issues.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Known compatibility issues with Android apps
+sidebar_position: 14
+---
+
+## Conflict apps
+
+### Various VPN apps
+
+All VPN apps are incompatible with AdGuard running in VPN filtering mode since you cannot have two active VPNs at the same time. To solve this problem we can suggest you:
+
+* to use AdGuard VPN that has a compatibility mode with AdGuard out-of-the-box
```

Comment:
* use AdGuard VPN that is compatible with AdGuard "out-of-the-box"
/
* use AdGuard VPN — it has Integrated mode that allows two apps to operate seamlessly